### PR TITLE
add README for jsk_fetch

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

File: jsk_fetch_robot/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+jsk_pepper_robot
```
Comment (kochigami) [inline jsk_fetch_robot/README.md]: Hi, I think it's not pepper, but fetch.

**Comment by itohdak** [comment]
```
send *ri* :angle-vector-raw (send *fetch* :angle-vector) 5000
```

File: jsk_fetch_robot/README.md (hunk 1 above)
Comment (k-okada) [inline jsk_fetch_robot/README.md]: thank you

**Comment by furushchev** [comment]
What is the relationship with this and the `README` on jsk_fetch_startup (https://github.com/jsk-ros-pkg/jsk_robot/tree/master/jsk_fetch_robot/jsk_fetch_startup )?

**Comment by Affonso-Gui** [comment]
Notes on execution and control of jsk_fetch

```
source devel/setup.bash
ping fetch15

source devel/setup.bash
rossetip
rossetmaster ***

roscd jsk_fetch_robot
cd launch
roslaunch rviz.launch

----
cat fetcheus/fetch.yaml
	"joint information"
	"e.g. torso-waist-z -> :torso :waist-z :joint-angle"

---------

roseus
(load "package://fetcheus/fetch-interface.l")
>> setq *fetch* (fetch)
>> objects *fetch*
>> setq *ri* (instance fetch-interface :init)

send *fetch* :angle-vector
	"Display *fetch*'s values"

send *fetch* :angle-vector #f(* * * * * * * * * *)

send *fetch* :angle-vector (send *ri* :state :potentio-vector)
	"Send initial position to *fetch*"

(send *fetch* :reset-pose)
	"Reset pose"

send *fetch* :torso :waist-z :joint-angle 200
	"torso-waist-z set in 200 [deg.cm]" 

send *ri* :angle-vector-raw (send *fetch* :angle-vector) 5000
	"Move to *fetch*'s position in 5000 [ms]"

```

**Comment by k-okada** [comment]
> What is the relationship with this and the README on jsk_fetch_startup (https://github.com/jsk-ros-pkg/jsk_robot/tree/master/jsk_fetch_robot/jsk_fetch_startup )?

is for admins, and this is for users

**Comment by k-okada** [comment]
I have updated the readme document, please look at https://github.com/k-okada/jsk_robot/blob/950c5a32dd0adbf168019abb195883dc3480dd05/jsk_fetch_robot/README.md

**Comment by furushchev** [comment]
@k-okada Thank you now it's clear. :+1:

**Pull request merged**